templates/_helpers.tpl

```diff
@@ -43,7 +43,7 @@ stackstorm
 {{- $template := index . 2 }}
 {{- $values := $dot.Values }}
 {{- range $subchart }}
-{{- $values = index $values . }}
+{{- $values := index $values . }}
```

Comment:
I believe existing code behaves as intended.
The variable was declared before, but gets re-assigned via `=` in the loop.

Also see https://golang.org/doc/go1.11#text/template

Reply:
it failed for me when I ran it with the error pointing to this line..  after i changed it, it worked.

Reply:
What was the Helm version you used?
Is there an exact error log you can share?

Reply:
Error: parse error in \"stackstorm-ha/templates/_helpers.tpl\": template: stackstorm-ha/templates/_helpers.tpl:46: unexpected \"=\" in operand"

Reply:
What's your `helm version` ?

Reply:
Client: &version.Version{SemVer:"v2.11.0", GitCommit:"2e55dbe1fdb5fdb96b75ff144a339489417b146b", GitTreeState:"clean"}

Reply:
Helm `v2.11.0` was released 1 year ago with K8s `v1.11` that reached EOL.

As I checked the closest Helm version that doesn't have templating issue you're experiencing is `v2.12.0`. 
If you can just upgrade to that next version, - that would fix it. Better yet - to latest Helm, if possible.